crates/driver/src/tests/setup/mod.rs

```diff
@@ -1175,8 +1208,14 @@ impl<'a> SolveOk<'a> {
                 Some(executed_amounts) => (executed_amounts.sell, executed_amounts.buy),
                 None => (quoted_order.sell, quoted_order.buy),
             };
-            assert!(u256(trade.get("sellAmount").unwrap()) == expected_sell);
-            assert!(u256(trade.get("buyAmount").unwrap()) == expected_buy);
+            assert!(is_approximately_equal(
+                u256(trade.get("sellAmount").unwrap()),
+                expected_sell
+            ));
+            assert!(is_approximately_equal(
+                u256(trade.get("buyAmount").unwrap()),
+                expected_buy
+            ));
```

Comment:
Had to use `is_approximately_equal` since now the actual scaling math is used for the first time. The consequence is that scaling two U256 numbers by the same `factor` doesn't mean those two numbers have the same proportion as before scaling, which may introduce rounding errors when the executed sell/buy amounts are calculated.